### ISSUE TITLE
Not able to build the project 

### ISSUE THREAD

**Issue by Dinakaranevil** [libbpf/libbpf-rs#142]
I have installed this library using git and wanted to run some scripts
```
 error[E0063]: missing fields `__bindgen_padding_0` and `__bindgen_padding_1` in initializer of `bpf_object_open_opts`
  --> /home/Desktop//libbpf-rs/libbpf-rs/src/object.rs:62:9
   |
62 |         libbpf_sys::bpf_object_open_opts {
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ missing `__bindgen_padding_0` and `__bindgen_padding_1`

For more information about this error, try `rustc --explain E0063`.
error: could not compile `libbpf-rs` due to previous error

```
i am getting this following error when i try to run the command cargo build

my Cargo.toml looks like this

```
package]
name = "my_project"
version = "0.1.0"
edition = "2018"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
anyhow = "1.0"
chrono = "0.4"
libbpf-rs = { path = "../libbpf-rs/libbpf-rs" }
libc = "0.2"
plain = "0.2"
structopt = "0.3"

[build-dependencies]
libbpf-cargo = { path = "../libbpf-rs/libbpf-cargo" }


```

**Comment by anakryiko** [comment]
Probably related to recent changes (https://github.com/libbpf/libbpf-rs/commit/ebc55bae01b9de644491f628ab95aeb1a4069edd). @mimullin-bbry any suggestions?

**Comment by mimullin-bbry** [comment]
From the look of his cargo.toml where he has the source code on his machine using path = "..", he might need to pull the change you listed.

However, we also see a similar issue where if we have a cargo toml of

[dependencies]
libbpf-rs = "0.13"

This will pull the "shipped" version of libbpf-rs (ie https://github.com/libbpf/libbpf-rs/commit/ec88b9caea7ef95d99ca209838dcafa6cf2187cc), but libbpf-sys of 0.5.0-2.

My theory is that cargo doesn't understand the - character of libbpf-sys, and thus the explicit statement of
libbpf-sys = "0.5.0-1" 
isn't properly understood by cargo, and it pulls the latest version of 0.5.0*

**Comment by mimullin-bbry** [comment]
To reproduce this issue follow the given steps
```

git clone https://github.com/libbpf/libbpf-rs.git libbpf-rs-test
cd libbpf-rs-test
git checkout 217d2986bb37ea3ecc9a671076cccf591c49c58f
rm Cargo.lock
cargo check
```

The change I requested (https://github.com/libbpf/libbpf-rs/commit/ebc55bae01b9de644491f628ab95aeb1a4069edd) solves the issue.

**Comment by anakryiko** [comment]
I just published libbpf-rs 0.14.0 and libbpf-cargo 0.9.3, please update and try if that fixes your issues. Thanks.

**Comment by mimullin-bbry** [comment]
both 
libbpf-rs = "0.14"
and
libbpf-rs = "*"

verified working for our project.  Thank you @anakryiko 

Note: You may wish to update the readme file and the description of the project on crates.io as both point to 0.11 and 0.8 for libbpf-rs and libbpf-cargo respectively.
